server/di/di.go

```diff
@@ -0,0 +1,68 @@
+package di
```

Comment:
nit: what does this stand for? can we use a more descriptive name for this package?